### ISSUE TITLE
Mooltipass device switch not recognized by app

### ISSUE THREAD

**Issue by NicoHood** [limpkin/mooltipass#235]
### Expected behavior
When disconnecting a MP Mini and connecting a normal MP it should be recognized by the app.

### Actual behavior
The app shows the settings for the MP Mini still even if a normal one is connected.

### Step by step guide to reproduce the problem
* Connect MP Mini
* Open App
* Go to Device Settings
* Connect normal MP 

### Mooltipass version
v61 beta and v1.0

**Comment by limpkin** [comment]
are you sure about that? I just tried it and the change correctly happens (even though it takes a few seconds).

**Comment by NicoHood** [comment]
You have to not focus the app. Just click somewhere else when you unplug the MP Mini and then replug the MP.

**Comment by limpkin** [comment]
yup, still can't reproduce the "issue"

**Comment by NicoHood** [comment]
You are right. It just sometimes happens. The actual bug is possibly that if you disconnect the mini the app will still show its device settings. So for some reason the disconnecting was missed. And so the new MP won't be recognized properly.

I am trying to search what exactly triggers the bug, but it might be just timing.

**Comment by NicoHood** [comment]
The bug also triggers that an unlocked MP is not recognized as unlocked. You only need 1 MP Mini to reproduce this. If the device settings tab stays open it got triggered. I had this 3 times now.

To reproduce the bug try to switch the tabs "device settings" and "credntials" while connecting the MP Mini. That made it quite easy to reproduce.

**Comment by limpkin** [comment]
ok I really can't reproduce this "issue"... could you make a video or look inside our source code?

**Comment by NicoHood** [comment]
I've made a video which I cannot send here. For documentation purposes I want to decribe what I did in the video:
* Open the App
* Connect MP Mini
* Go to Device settings (wrong settings displayed)
* Unlock MP Mini
* Disconnect, Connect normal MP (0:35)
* Correct Device Settings displayed
* Disconnect, connect MP Mini (0:50)
* Old MP Device Settings still loaded
* Unlocked MP Mini
* Device Settings are correct again
* Disconnect, Connect old MP (1:15)
* Disconnect,  Connect  MP Mini.
* The old settings are still displayed (where I moved the window)

It possibly happens if you connect the device too fast, so the disconnect is not recognized. That the MP Mini settings are not displayed when unlocked might be a feature, however if so it should say "locked" and not some cz layout.

Please contact me in tox to send you the video. Its 7mb.

**Comment by limpkin** [comment]
If you're saying that the settings aren't changing until the mooltipass (standard or mini) is unlocked... then it is normal as the firmware isn't responding to usb packets during the unlocking process.

**Comment by NicoHood** [comment]
No, watch the video. You cannot see that I unplug the device, but that is actually the bug. Its not recognized anymore. contact me tomorrow and I will send you the video.

If the device is locked however the menu should be disabled and not display wrong values. That is possibly another bug.

**Comment by NicoHood** [comment]
Do you still want the video? I had this issue so many times now. Its 100% reproduceable for me. Or should I rather try with moolticute when its ready?

**Comment by limpkin** [comment]
it's in our todo list to disable the settings tab until the selected settings is correctly applied

**Comment by limpkin** [comment]
alternatively, you can make a PR which implements the previously described functionality.
